pkg/controller/embercsi/controller.go

```diff
@@ -76,7 +76,7 @@ func getControllerContainers(ecsi *embercsiv1alpha1.EmberCSI) []corev1.Container
 			Image: Conf.Sidecars[Cluster].Attacher,
 			Args: []string{"--v=5",
 				"--csi-address=/csi-data/csi.sock",
-				"--connection-timeout=120s",
```

Comment:
External-attacher v2.0 only works for K8s v1.14 and later, so we would need to maintain the old parameter for those versions  right?

Reply:
Right, but since time has passed by I think this should be safe today. Wdyt?

Reply:
Nevermind, this has been replaced with the general timeout setting, dropped the commit.